st/initscripts

```diff
@@ -27,12 +28,61 @@ set -e
 M0_SRC_DIR=`readlink -f $0`
 M0_SRC_DIR=${M0_SRC_DIR%/*/*}
 
-. $M0_SRC_DIR/utils/functions # report_and_exit
+
+echo $M0_SRC_DIR
```

Comment:
This echo command is not needed. This can be removed.

Reply:
Okay I will update it.